database/seed.js

```diff
@@ -0,0 +1,97 @@
+const db = require('./index.js');
+
+
+// bundle
+// for each bundle 1-100
+// 3 random tiers
+// random cost 5-20
```

Comment:
Nice use of comments to set expectations of developers reading your code.  Concise but meaningful.